securityscanutils/issuewriter/local_writer.go

```diff
@@ -0,0 +1,47 @@
+package issuewriter
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/google/go-github/v32/github"
+)
+
+type LocalIssueWriter struct {
+	repo      GithubRepo
+	outputDir string
+}
+
+var liw IssueWriter = &LocalIssueWriter{}
+
+func NewLocalIssueWriter(repo GithubRepo, outputDir string) (IssueWriter, error) {
+	// Set up the directory structure for local output
+	err := os.MkdirAll(outputDir, os.ModePerm)
+	if err != nil {
+		return nil, err
+	}
+	return &LocalIssueWriter{
+		repo:      repo,
+		outputDir: outputDir,
+	}, nil
+}
+
+func (l *LocalIssueWriter) Write(ctx context.Context, release *github.RepositoryRelease, contents string) error {
+	version, err := semver.NewVersion(release.GetTagName())
+	if err != nil {
+		return err
+	}
+	filename := path.Join(l.outputDir, version.String()+".md")
+	f, err := os.OpenFile(filename, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintf(f, contents)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
What if we just conslidate this to? (happy to leave as is, also):
```suggestion
	return err
```

Reply:
I typically use this pattern for consistency and to be fully explicit and avoid mixing return patterns. In writing this PR, I almost introduced an issue (luckily caught by a unit test) where I mistakenly wrapped a `return err` with `return eris.Wrapf(MyError, err.Error())` because I saw a `return err`, but that `err` value was actually `nil`.